### DISCUSSION TITLE
First post title

### DISCUSSION THREAD

**Discussion by alabuzhev** [FarGroup/FarManager#312] [General]
First post text

**Comment by nightroman** [comment]
The new Discussions feature looks promising. It would be nice to have the official guidelines, e.g in README.md, when and what to post here vs. on forums.

**Reply by alabuzhev** [reply]
I'd say it's the same as with [issues](https://github.com/FarGroup/FarManager/issues) vs [bugs](https://bugs.farmanager.com): forums are preferable in general, but this one will do if you don't feel like registering there for a quick question.

**Reply by johnd0e** [reply]
Or in case if forum is down (like now).

**Comment by johnd0e** [comment]
Is English demanded here?
If not - then it make sense to add Russian welcome topic too.
